src/nonprinting_notation.rs

```diff
@@ -0,0 +1,11 @@
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum NonprintingNotation {
+    Caret,
+    Unicode,
+}
+
+impl Default for NonprintingNotation {
+    fn default() -> Self {
+        NonprintingNotation::Unicode
+    }
+}
```

Comment:
Can you add docs and automatically derive Default please? Proposal:

```suggestion
/// How to print non-printable characters with
/// [crate::config::Config::show_nonprintable]
#[derive(Debug, Default, Clone, Copy, PartialEq, Eq)]
pub enum NonprintingNotation {
    /// Use caret notation (^G, ^J, ^@, ..)
    Caret,

    /// Use unicode notation (␇, ␊, ␀, ..)
    #[default]
    Unicode,
}
```